Gemfile

```diff
@@ -77,7 +77,7 @@ group :test do
   gem 'factory_girl_rails'
 
   # database cleaner
-  gem 'database_rewinder'
+  gem 'database_rewinder', github: 'amatsuda/database_rewinder'
```

Comment:
We do not recommend referencing master because it is in a state that has not been released yet. The fact that it is not released is because master is likely to change more and more.

If you do, you'd better refer to what you fork to shinosakarb.

FYI
https://twitter.com/sue445/status/818989506510475264